src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -125,11 +125,11 @@ def acs_browse(cmd, client, resource_group, name, disable_browser=False, ssh_key
 def _acs_browse_internal(cmd, client, acs_info, resource_group, name, disable_browser, ssh_key_file):
     orchestrator_type = acs_info.orchestrator_profile.orchestrator_type  # pylint: disable=no-member
 
-    if orchestrator_type == 'kubernetes' or \
+    if str(orchestrator_type).lower() == 'kubernetes' or \
```

Comment:
What type could this `orchestrator_type` to be? Why does it need to be cast into `str` first?

Reply:
It could be an instance of `azure.mgmt.containerservice.models.ContainerServiceOrchestratorTypes` at runtime, which is just an enum class that can be cast to `str` but isn't directly comparable.